api/src/pcapi/core/finance/models.py

```diff
@@ -72,7 +72,21 @@ class Deposit(PcObject, Base, Model):  # type: ignore [valid-type, misc]
     def specific_caps(self):  # type: ignore [no-untyped-def]
         from . import conf
 
-        return conf.SPECIFIC_CAPS[self.type][self.version]
+        PHYSICAL_CAP = None
+        DIGITAL_CAP = None
+
+        if self.type == DepositType.GRANT_18:
+            if self.version == 1:
+                PHYSICAL_CAP = conf.GRANT_18_PHYSICAL_CAP_V1
+                DIGITAL_CAP = conf.GRANT_18_DIGITAL_CAP_V1
+            elif self.version == 2:
+                PHYSICAL_CAP = conf.GRANT_18_PHYSICAL_CAP_V2
+                DIGITAL_CAP = conf.GRANT_18_DIGITAL_CAP_V2
+
+        if self.user.departementCode == conf.WALLIS_AND_FUTUNA_DEPARTMENT_CODE:
+            DIGITAL_CAP = None
+
+        return conf.SpecificCaps(digital_cap=DIGITAL_CAP, physical_cap=PHYSICAL_CAP)
```

Comment:
:rotating_light: PEP8 POLICE, DON'T MOVE :rotating_light: 

C'est quoi toutes ces majuscules ? :scream_cat: 

```python
physical_cap = None
digital_cap = None

# etc.
```

Reply:
:scream: 
OMG officer I have no idea how it got there !

Je change ça immédiatement, bad copy pasting

Reply:
![Under arrest](https://c.tenor.com/2sBSdgRDSgIAAAAC/under-arrest-youre-under-arrest.gif)